vcr/verifier/verifier_test.go

```diff
@@ -558,31 +333,32 @@ func TestVerifier_VerifyVP(t *testing.T) {
-			const keyID = "did:key:z6MksEyx6d5pB1ekoaVmaGsibbcYHE9Vxx7V13PSqPwxYRzL#z6MksEyx6d5pB1ekoaVmaGsibbcYHE9Vxx7V13PSqPwxYRzL"
-			keyAsJWK, err := jwk.ParseKey([]byte(`{
-        "kty": "OKP",
-        "crv": "Ed25519",
-        "x": "vgLDESnU0TIlW-PmajyrvSlk9VysAsRkSYiEPBELj-U"
-      }`))
-			require.NoError(t, err)
-			require.NoError(t, keyAsJWK.Set("kid", keyID))
-			publicKey, err := keyAsJWK.PublicKey()
-			require.NoError(t, err)
-
-			presentation, err := vc.ParseVerifiablePresentation(rawVP)
-			require.NoError(t, err)
-			ctx := newMockContext(t)
-			ctx.keyResolver.EXPECT().ResolveKeyByID(keyID, gomock.Any(), resolver.NutsSigningKeyType).Return(publicKey, nil)
-
-			vcs, err := ctx.verifier.VerifyVP(*presentation, false, false, nil)
 
-			assert.EqualError(t, err, "verification method is not of issuer")
-			assert.Empty(t, vcs)
-		})
+		//t.Run("VP signer != VC credentialSubject.id", func(t *testing.T) {
```

Comment:
@reinkrul credentialSubject.id is no longer used to derive the signer of a VP. What should happen to this test? also see #2684

Reply:
added check to VerifyVP and re-enabled test